content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,30 @@ menu:
     parent: About the project
 ---
 
+<!--  ## v1.8.4 [unreleased]
+
+The InfluxDB Enterprise 1.8.4 release builds on the InfluxDB OSS 1.8.4 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v1-8-4-unreleased).
+
+### Bugfixes
+
+- Fix "snapshot in progress" error that occurred during backup.
```

Comment:
This bug fix will not be in 1.8.4.

Reply:
thanks, David, yes, I hid this bit, so we can have it for next time

Reply:
should clarify, anywhere you see text start 
`<!--
...
-->`
something has been intentionally hidden